### PR TITLE
{tools}[GCC/4.9.2] iRODS 4.0.3 i-commands (REVIEW)

### PR DESCRIPTION
**Pull request by martbhell**
Hello from the 9th Hackathon

### CODE REVIEW

**Comment by martbhell** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by martbhell** [comment]
The i-commands built here do not include GSS/Globus support.

File: easybuild/easyconfigs/i/iRODS/iRODS-4.0.3-GCC-4.9.2.eb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# are installed.
+cmds_vars = 'EBROOTIRODS=%(installdir)s/lib LIBCURLDEV=y PYTHONDEV=y BZIP2DEV=y'
+cmds_str = cmds_vars + ' ./packaging/build.sh icommands -r' 
+cmds_map = [('', cmds_str)]
+
+# irods_build_sh.patch removes build.sh dependency checks (find /usr curl.h etc)
+# of python-devel, curl-devel and bzip2-devel (as these are in easybuild).
+# irods_find_irods_home.patch alters the path returned by finds_irods_home.sh to
+# installdir/lib
+patches = ['irods_build_sh.patch', 'irods_find_irods_home.patch']
```
Comment (boegel) [inline easybuild/easyconfigs/i/iRODS/iRODS-4.0.3-GCC-4.9.2.eb]: include version in names of patch files?
also, we generally use the actual software name, like:

```python
patches = [
    'iRODS-%(version)s_build_sh.patch',
    'iRODS-%(version)s_irods_find_irods_home.patch',
]
```

**Comment by migueldiascosta** [comment]
@martbhell, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**